client/my-sites/jetpack-search/purchases.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import { isJetpackSearch, planHasJetpackSearch } from '@automattic/calypso-products';
+
+export const hasJetpackSearchPurchaseOrPlan = (
+	sitePurchases: Record< string, unknown > | null,
+	planSlug: string | null
+): boolean => {
+	if ( ! sitePurchases || ! planSlug ) {
+		return;
```

Comment:
```suggestion
		return false;
```
The return type is set to boolean, so I guess we'd better return one.

Reply:
Ah! It used to be the case that you could return `null` for any type, but that seems to no longer be the case in TS 2.0+:

https://stackoverflow.com/questions/42533120/typescript-define-type-that-can-be-bool-or-null